src/MyHealth/nativeModule.ts

```diff
@@ -0,0 +1,55 @@
+import { NativeModules } from "react-native"
```

Comment:
Nice, thanks for pulling this module out.

Reply:
💯  credit to you